spack/package.py

```diff
@@ -93,18 +95,20 @@ class Arbor(CMakePackage, CudaPackage):
     depends_on("py-mpi4py", when="+mpi+python", type=("build", "run"))
 
     # python (bindings)
-    extends("python", when="+python")
-    depends_on("python@3.7:", when="+python", type=("build", "run"))
-    depends_on("py-numpy", when="+python", type=("build", "run"))
     with when("+python"):
+        extends("python")
+        depends_on("python@3.7:", type=("build", "run"))
```

Comment:
This is still 3.7, why?

Reply:
This is pre-Arbor 0.9.1, so any version released up to now.